src/app/utils/constants.ts

```diff
@@ -0,0 +1,8 @@
+export const CONTRACT_ADDRESS = "0x40f72B6ac30bca6Bdc67321F7706406871C7E36c";
+export const MULTICALL_ADDRESS = "0xca11bde05977b3631167028862be2a173976ca11";
```

Comment:
FYI - multicall is already defined in the skale chain object and can be picked from there